Flagsmith.FlagsmithClient/Interfaces/ITrait.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Flagsmith.Interfaces
+{
+    public interface ITrait
+    {
+        string Key { get; }
+        dynamic Value { get; }
+
+        [Obsolete("Use 'Key' property, method will be removed in next version.")]
```

Comment:
Again, we could probably remove these methods / obsolete statements?

Reply:
same here...